easybuild/easyblocks/t/tensorflow.py

```diff
@@ -125,7 +125,7 @@ def configure_step(self):
         if cuda_root:
             config_env_vars.update({
                 'CUDA_TOOLKIT_PATH': cuda_root,
-                'GCC_HOST_COMPILER_PATH': which(os.getenv('CC')),
+                'GCC_HOST_COMPILER_PATH': os.path.realpath(which(os.getenv('CC'))),
```

Comment:
@smoors Please use the `resolve_path` function from `filetools` (which has error handling)